pyseir/inference/initial_conditions_fitter.py

```diff
@@ -229,6 +229,7 @@ def _fit_fips(fips, generate_report=False):
         return {"model_params": None, "t0_date": None, "reduced_chi2": None}
 
 
+# Chris: Can we delete??? Not used anywhere and I don't think start dates matter...
```

Comment:
Remove this whole file (InitialConditionsFitter class) and the cli calls associated with this. It isn't used anymore.